isp/.openapi-generator-ignore

```diff
@@ -32,3 +32,4 @@ go.sum
 git_push.sh
 
 convenience.go
+client.go
```

Comment:
Does your change require this?

The previous PR moved this file to the `prerequisites` [model that preserves the file](https://github.com/istreamlabs/go-sdk/pull/20/files#diff-0a1a03204f56df2e413b62ff5c8a200525fd08d17fea9428324850013f35855fR35), then copies it in after the generator is run.

I'm not 100% sure of the context, but I think that lets us see when the generator makes changes to `client.go` because of changes in the OpenAPI spec, but also lets us maintain any manual modifications made to the file--if the generator shows changes, we copy the manual modifications into the new `client.go` and move it to the `prerequisites` directory.

Reply:
That makes sense. I saw the new prerequisites folder but didn't put two and two together about the ignores. I've removed this and am honestly not sure what added it, but I don't need it for this proposed PR change.